tests/snuba/tagstore/test_tagstore_backend.py

```diff
@@ -212,6 +212,31 @@ def test_get_tag_keys(self):
         }
         assert set(keys) == expected_keys
 
+    def test_get_tag_keys_with_denylist(self):
+        denylist_keys = frozenset(["browser"])
+        expected_keys = {
+            "baz",
+            "environment",
+            "foo",
+            "sentry:release",
+            "sentry:user",
+            "level",
+        }
+        keys = {
+            k.key: k
+            for k in self.ts.get_tag_keys(
+                project_id=self.proj1.id, environment_id=self.proj1env1.id, denylist=denylist_keys
+            )
+        }
+        assert set(keys) == expected_keys
+        keys = {
+            k.key: k
+            for k in self.ts.get_tag_keys(
+                project_id=self.proj1.id, environment_id=self.proj1env1.id, denylist=denylist_keys
+            )
+        }
+        assert set(keys) == expected_keys
```

Comment:
What is the point of this portion of the test?

Reply:
to hit 100% code coverage? but i assume another test can do it right?

Reply:
yeah i copied it from another test, 100% we can simplify this test.

Reply:
So I'd recommend testing within this test once with the deny list and once without for completeness

Reply:
fixed in fac7ee5bf8